src/plugins/dna_export/src/ExportMSA2MSADialog.h

```diff
@@ -44,6 +44,19 @@ class ExportMSA2MSADialog : public QDialog, Ui_ExportMSA2MSADialog {
         Gap
     };
 
+    /*!
+     * \class TranslationFrame
+     * \brief Frame to translate (reverse-complement if minus)
+     */
+    enum class TranslationFrame {
```

Comment:
Do we want to introduce such enum in every task/dialog where translation frames are used? 
If yes: this is overdesign. If not: either add the enum to some common core header or use simple 1,2,3,-1,-2,-3 in this class as we did before in other places.

If you will decide to add the enum please use more meaningful names for the fields: FirstDirectFrame FirstReverseComplementFrame or similar.

Naming "-1" as MinusOne makes no value.

Reply:
I decided to get rid from the enum